cogs/welcome/welcome.py

```diff
@@ -537,6 +644,104 @@ def check(message: discord.Message):
         await self.config.guild(ctx.guild).get_attr(key).set(greetings)
         return
 
+    # [p]welcomeset greetings image
+    @greetings.group(name="image")
+    async def image(self, ctx: Context):
+        """Base command for the image command group"""
+        pass
+
+    # [p]welcomeset greetings image template
+    @image.command(name="template")
+    async def imageTemplate(self, ctx: Context):
+        await ctx.send(
+            "Here is the welcome image template so you can make your own! For best results please render the image at 72dpi, 1193 x 671. The bot will try to make it conform automatically but mileage may vary.",
+            file=discord.File(
+                os.path.join(data_manager.bundled_data_path(self), "welcome_template.png")
```

Comment:
Here and elsewhere there are constants being used. Can we declare extract them to constants in `constants.py`?